app/health.go

```
@@ -48,7 +48,7 @@ func NewHealthContext(c *cli.Context) *HealthContext {
 	return context
 }
 
-func simpleHealthCheck(c *cli.Context) {
+func simpleHealthCheck(c *cli.Context) error {
```

I don't think we need to change the signature, since it Fatals out, the err is never returned.